ci/do_ci.sh

```diff
@@ -34,8 +33,8 @@ elif [[ "$1" == "bazel.asan" ]]; then
   cd "${ENVOY_CONSUMER_SRCDIR}"
   echo "Building and testing..."
   # TODO(htuch): This should switch to using clang when available.
-  bazel --batch test ${BAZEL_TEST_OPTIONS} -c dbg --config=asan --test_output=all \
-    --cache_test_results=no @envoy//test/... //:echo2_integration_test
+  bazel --batch test ${BAZEL_TEST_OPTIONS} -c dbg --config=asan @envoy//test/... \
```

Comment:
I'm a bit confused, which of the labels do you want me to prefix with @envoy? @envoy is being used here to refer to the `local_repository`.